modules/tinymce/src/plugins/table/test/ts/browser/ClipboardTest.ts

```diff
@@ -678,4 +680,56 @@ describe('browser.tinymce.plugins.table.ClipboardTest', () => {
       '</table>'
     );
   });
+
+  it('TINY-7485: Pasting into a table with a single cells contents selected', () => {
```

Comment:
This test label seems wrong. Maybe `Pasting into a table with single cells contents selected` (without `a`)?!

Reply:
The a is correct

Reply:
This might help

```suggestion
  it('TINY-7485: Pasting into a table with a single cell's contents selected', () => {
```